### PR TITLE
PT-607 DBLHU Level 1 & 2 parameters

### PR DESCRIPTION
**Pull request by CedricPerotto**

### CODE REVIEW

File: store/store_test.go [hunk 1]
```diff
@@ -1196,3 +1196,40 @@ func TestStore_LatestDeviceIdFilter(t *testing.T) {
...
+			},
+		})
+	var res string
+	var err error
+	if res, err = store.GetDeviceModel("dblg1_1"); err != nil {
+		t.Error("Unexpected Error during device model request")
+	}
+	// Retreiving latest (time field desc) payload.device.name not null value
+	if res != "DBLG1" {
+		t.Error("Unexpected device model calue")
```
Comment (franck-fourel) [inline store/store_test.go]:
```suggestion
		t.Errorf("%s should be equal to DBLG1", res)
```

File: store/store_test.go [hunk 2]
```diff
@@ -1196,3 +1196,40 @@ func TestStore_LatestDeviceIdFilter(t *testing.T) {
...
+			"payload": bson.M{
+				"device": bson.M{
+					"name": "DBLG1",
+				},
+			},
+		})
+	var res string
+	var err error
+	if res, err = store.GetDeviceModel("dblg1_1"); err != nil {
+		t.Error("Unexpected Error during device model request")
```
Comment (franck-fourel) [inline store/store_test.go]:
```suggestion
		t.Errorf("Unexpected Error during device model request: %s", err)
```

File: tide-whisperer.go [hunk 3]
```diff
@@ -296,6 +296,16 @@ func main() {
...
 			defaultLevelFilter := make([]int, 1)
 			defaultLevelFilter = append(defaultLevelFilter, 1)
+
+			var device string
+			var deviceErr error
+			if device, deviceErr = storage.GetDeviceModel(queryParams.UserId); deviceErr != nil {
+				log.Printf("Error in GetDeviceModel for user %s. Error: %s", queryParams.UserId, deviceErr)
+			}
+			if device == "DBLHU" {
+				defaultLevelFilter = append(defaultLevelFilter, 3)
```
Comment (franck-fourel) [inline tide-whisperer.go]: why is defaultLevelFilter set to 3? 
shouldn't be 2?

Comment (CedricPerotto) [reply tide-whisperer.go]: So called "Level 2 parameters" are stored with a level 3 value...

**Pull request merged**